helm/osm-seed/values.yaml

```diff
@@ -4,18 +4,21 @@
 # ====================================================================================================
 # ====================================================================================================
 # The version of the image group in osm-seed, get it here: https://hub.docker.com/r/developmentseed/osmseed-web/tags/
-osmSeedVersion: cloudProvider-b1fee6e
+osmSeedVersion: tiler-6be452f
```

Comment:
@Rub21 can we try and avoid making these kinds of changes in the default values file, and leave default pointed to `master` or so? And just over-ride locally?

Reply:
@batpad , ok, once I merge these changes it will generate a new one and I will update with that value..